cli.js

```diff
@@ -41,6 +42,18 @@ var conf = pkgConf.sync('ava', {
 	}
 });
 
+// check for valid babel config shortcuts (can be either "default" or "inherit")
+var isValidShortcut = ['default', 'inherit'].indexOf(conf.babel) >= 0;
+
+if (conf.babel.length > 0 && !isValidShortcut) {
```

Comment:
That way it will fail when `babel` config is not even present. It should only check and display this error message, when `babel` config is there and has invalid value.